streaming/java/streaming-api/src/main/java/io/ray/streaming/api/function/Function.java

```diff
@@ -7,4 +7,12 @@
  */
 public interface Function extends Serializable {
 
+  default void loadCheckpoint(Object checkpointObject) {
+
+  }
+
+  default Object doCheckpoint() {
```

Comment:
ditto. Why use Object. For such an important method, we should give a more specified type

Reply:
I'' use Serializable instead

Reply:
done.